README.rst

```diff
@@ -54,7 +54,7 @@ within the CMake source directory or any other build directory of your
 choice. Once this has finished successfully, run ``make`` and
 ``make install``.  In summary::
 
- $ ./bootstrap && make && make install
+ $ ./bootstrap && gmake && gmake install
```

Comment:
@Kentzo OK, I'll change README back. Please note that I really used `gmake` for build process.
Also I was able to build CMake under Windows. With VS2015. This needs CMake for Windows and some patches to files (include windows-specific headers and so on).